working-open.md

```diff
@@ -0,0 +1,46 @@
+# Working Open
+
+Hypha operates with democratic organizing principles informed by our [values](./values.md). As part of that we've developed a set of guidelines for how (and exactly what) and why we work in the open.
+
+## Why?
+
+- In order to share, provide opportunities for self- and collective- learning as we work alongside other new economy projects.
+- Not because a "default to open" is an ideal, but because it orients us toward negotiations a set of practices around our shared intentions.
+- Yet we recognize openly sharing might prevent frank and necessary discussion beneficial to our members, and so we want to clearly lay our our practices of consent and privacy
+
+# How and what?
+
+- We currently work with a [*in camera*](https://en.wikipedia.org/wiki/In_camera)/redaction request for meeting notes, where at any point members can request to stop taking notes, or redact and correct notes after they have been taken
+
+Legend:
+👀 - View only
+
+**Default open to all**:
+
+- Meeting notes (all hands, working groups) and final documents (handbook), on GitHub and other platforms 👀
+- Aggregated financal statements 👀
+- Project deliverables*, task tracking, meeting notes, and aggregated financial statements 👀
+
+* *This would include source repositories and code (e.g., on GitHub) and other process materials based on consensus on licensing and usage constraints.*
+
+**Default open to members**:
+
+- Meeting notes and documents (write-access)
+- Task tracking, project management (PM), and decision making (DM) tools (write-access)
+- Finances (budget, expenses, and cash flow projections) and audit records 👀
+- Potential opportunities and leads
+- Project proposal(s), statement(s) of work (in-progress/signed), in-progres deliverables, and finances (budget, expenses, and cash flow projections) 👀
+
+**Default closed to working groups and project members**:
+
+- Full personal/financial information of members
```

Comment:
Does this reflect a discussion about sharing salaries internally? As in, do we want to explicitly refer to salaries in this doc? (I missed one of these all-hands meetings)

Reply:
I don't think we have had a discussion about "sharing" salaries but we have had a discussion about setting salaries floors as well as a max pay differential ratio (e.g., folks paid minimum XX/hour; max difference 1:2), I don't think it would be unreasonable to be very explicit about salaries and our pay guidelines, but that is different than actually sharing someone's YTD earnings.

My intent with this line (and I think a reflection of discussion about personal and financial information) what that it was for things like: home addresses, bank accts, SIN, health stuff (say if we have benefits and there is some minimum coordination of knowing OHIP details, or say a leave, or terrible case someone gets injured on the job... not like actually anyones health data), or also potential "HR" info. 

Some (scant) links on this: 
https://www.priv.gc.ca/en/privacy-topics/privacy-at-work/
https://www.minkenemploymentlawyers.com/employment-law-issues/workplace-privacy-an-increasingly-important-issue-in-the-information-age/

Reply:
Ah cool! Thanks! Adding that "for example" bit would def clarify this

Reply:
added!